aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -96,7 +117,9 @@ module AptosFramework::Coin {
 
     // Public functions
 
-    /// Burn coin with capability.
+    /// Burns a given amount of coins. This requires:
+    /// (1) The signing account to have burn capability (a resource created during the coin's initialization).
+    /// (2) The coins to burn are supplied to the function.
```

Comment:
This is true now but not true once we actually implement useful capabilities, Bors PR is the likely direction.

Reply:
Updated